### PR TITLE
Use FeeCalculator

### PR DESCRIPTION
**Pull request by jackcmay**

### CODE REVIEW

**Comment by jackcmay** [comment]
@jstarry I'm committing this to avoid rebase hell, please post-review at your leisure.

File: src/server/message-feed.js [hunk 1]
```diff
@@ -94,10 +96,14 @@ export default class MessageFeedController {
     console.log(`Reading ${elfFile}...`);
     const elfData = await fs.readFile(elfFile);
 
-    console.log('Loading program...');
+    console.log('Loading Message feed program...');
+    const [, feeCalculator] = await this.connection.getRecentBlockhash();
+    const fees =
+      feeCalculator.lamportsPerSignature *
+      (BpfLoader.getMinNumSignatures(elfData.length) + NUM_RETRIES);
```
[the remark this answers is not in the document]
Comment (jstarry) [reply src/server/message-feed.js]: Seems like maybe we should add a method to BpfLoader for estimating fees, what do you think @jackcmay?

[the remark this answers is not in the document]
Comment (jackcmay) [reply src/server/message-feed.js]: Yup, did add a method to estimate fees: `getMinNumSignatures()`.  But I'm seen where during loader there are sometimes `AccountInUse` or other errors and the fee's get charged.  I haven't dug into it deeply so in the examples I added some wiggle room for now.

[the remark this answers is not in the document]
Comment (jstarry) [reply src/server/message-feed.js]: Ah ok, sounds good!

**Pull request merged**